### PR TITLE
Add article Playwright for Python

### PR DESCRIPTION
**Pull request by dino-vinkovic**
Added a simple and short article with instructions for getting stared with Playwright for Python.
The setup is complete, it includes everything from the first step (installation) to the last one (successfully running the tests).

Much like the Puppeteer example, I would not go into more detail for now. We can expand on it later on when / needed.

### CODE REVIEW

File: Automation/Web/Get started with Playwright for Python.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+## Setup
+
+1. Create a new PyCharm project 
+   * Create and activate a [virtual environment](https://infinum.com/handbook/qa/automation/python/virtual-environment)
+2. Install [pytest-playwright](https://pypi.org/project/pytest-playwright/) plugin 
+   * `pip install pytest-playwright`
+3. Install browsers 
+   * `playwright install`
+4. Create a test file within the current working directory or a sub-directory
+   * File name e.g.: `test_homepage.py `
```
Comment (igor-pekovic) [inline Automation/Web/Get started with Playwright for Python.md:24]: There is an empty space in the end of the quote `test_homepage.py `

File: Automation/Web/Get started with Playwright for Python.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+   * See example test below
+5. Run the test
+   * On default (headless) browser:
+     * `python3 -m pytest test_homepage.py`
+   * On specific (headed) browser:
+     * `python3 -m pytest test_home_page.py --browser firefox –headed`
```
[the remark this answers is not in the document]
Comment (igor-pekovic) [reply Automation/Web/Get started with Playwright for Python.md:30]: Shouldn't the `headed` argument be preceded with a double hyphen `--`?

File: Automation/Web/Get started with Playwright for Python.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+   # Click the button
+   accept_all_button.click()
+
+   # Expect the button to be hidden
+   expect(accept_all_button).to_be_hidden()
+
+   # Take a full page screenshot (screenshot of a full scrollable page)
+   page.screenshot(path="screenshot_full_page.png", full_page=True)
+```
```
Comment (igor-pekovic) [inline Automation/Web/Get started with Playwright for Python.md:61]: Does the test run too quickly for you too? 😄 
I don't see it actually clicking Accept all button, it's like it doesn't wait for the button to be clickable 😅

File: Automation/Web/Get started with Playwright for Python.md [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+* Python 3.7 or newer.
+* Check [Playwright troubleshooting](https://playwright.dev/python/docs/troubleshooting) for more details.
+
+
+## Setup
+
+1. Create a new PyCharm project 
+   * Create and activate a [virtual environment](https://infinum.com/handbook/qa/automation/python/virtual-environment)
+2. Install [pytest-playwright](https://pypi.org/project/pytest-playwright/) plugin
```
Comment (igor-pekovic) [inline Automation/Web/Get started with Playwright for Python.md:19]: Can we remove slashes `/` from the end of URLs? 
It looks prettier when looking at the code 😄

**Review by igor-pekovic — CHANGES_REQUESTED** [review]

File: Automation/Web/Get started with Playwright for Python.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (dino-vinkovic) [reply Automation/Web/Get started with Playwright for Python.md:30]: Nice. Yes it should 😅 
It was copied from a file that merged the two dashes.

Tnx

File: Automation/Web/Get started with Playwright for Python.md (hunk 4 above)
Comment (dino-vinkovic) [reply Automation/Web/Get started with Playwright for Python.md]: Yep, not needed. Removed.

File: Automation/Web/Get started with Playwright for Python.md [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+## Example
+
+```python
+import re
+
+from playwright.sync_api import Page, expect
+
+
+def test_homepage(page: Page):
+   page.goto("https://infinum.com/")
```
Comment (igor-pekovic) [inline Automation/Web/Get started with Playwright for Python.md:42]: Same as [here](https://github.com/infinum/qa-handbook/pull/156#discussion_r1149670779).

**Review by igor-pekovic — APPROVED** [review]
I've tried it out, it works and it's fast 😄 

I just left one more tiny comment about formatting.

**Review by tividako — APPROVED** [review]

**Pull request merged**